clients/unshaded/src/test/java/tachyon/client/file/options/CompleteFileOptionsTest.java

```diff
@@ -15,34 +15,13 @@
 
 package tachyon.client.file.options;
 
-import java.util.Random;
-
 import org.junit.Assert;
 import org.junit.Test;
 
-import tachyon.conf.TachyonConf;
-
 /**
  * Tests for the {@link CompleteFileOptions} class.
  */
 public class CompleteFileOptionsTest {
-
-  /**
-   * Tests that building a {@link CompleteFileOptions} works.
-   */
-  @Test
-  public void builderTest() {
```

Comment:
For the sake of test coverage, we want something to ensure (1) the passed value is set to the right field (2) the returned value is `this`. what do you think?